src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        String[] resultArray;
```

Comment:
You don't need this variable. Use Collections.sort() for sorting and return the result by return result.toArray(new String[0])

Reply:
done